leaf-ui/rappid-extensions/BackboneModelsIntention.js

```diff
@@ -7,61 +8,339 @@
 
 /** 
  * Backbone Model of An Atomic Function Segment
+ * 
+ * @param {String} type
+ * Atomic function types
```

Comment:
Add the types. is it `I,C,D,R' or longer names?

Reply:
It's all of the functions types